app/Models/User.php

```diff
@@ -16,11 +16,7 @@ class User extends Authenticatable
      *
      * @var array
      */
-    protected $fillable = [
-        'name',
-        'email',
-        'password',
-    ];
+    protected $gaurded = [];
```

Comment:
First off, I know guarded provides you an easy way of mass assigning but then you might have issues when trying to update some guarded properties later on. You might result to having forceFills in your codebase which is not the best idea.

A Hacker can also mass update some columns that are not guarded in your database say ```is_admin``` column which is not supposed to be.

Reply:
Fixed